ui/src/shared/utils/formatStatValue.ts

```diff
@@ -45,7 +45,7 @@ export const formatStatValue = (
     return 'Data cannot be displayed'
   }
 
-  localeFormattedValue = preventNegativeZero(localeFormattedValue)
+  localeFormattedValue = '' + preventNegativeZero(localeFormattedValue)
```

Comment:
Why do we need this as a string? The return value of this function is a different variable which is already a string.